src/main/kotlin/app/widgets/welcomeBack/WelcomeBack.kt

```diff
@@ -0,0 +1,55 @@
+package app.widgets.welcomeBack
+
+import app.ui.styles.ButtonStyles
+import de.jensd.fx.glyphs.materialicons.MaterialIcon
+import de.jensd.fx.glyphs.materialicons.MaterialIconView
+import javafx.geometry.Pos
+import javafx.scene.layout.HBox
+import javafx.scene.paint.Color
+import javafx.scene.text.FontWeight
+import tornadofx.*
+import app.widgets.profileIcon.ProfileIcon
+import java.awt.Button
+import java.util.*
+
+class WelcomeBack : HBox() {
+    private val rad = 125.0
+    val messages = ResourceBundle.getBundle("Messages")
```

Comment:
shouldn't tornadofx be doing this for you though?

Reply:
oh, inside the widgets I wasn't able to reference the bundled resources. i'm not sure as to why? I think it has to do with how/where the bundles are configured.